### ISSUE TITLE
a11y: missing labels on controls (text fields, select boxes)

### ISSUE THREAD

**Issue by VladimirAus** [swagger-api/swagger-ui#6512]
Swagger UI renderer breaks accessibility rules: WCAG 1.3.1, WCAG 4.1.2

- path: <select><option value="http://localhost:1723">http://localhost:1723</option></select>
- also text fields, checkboxes etc

Fix ONE of the following:
- aria-label attribute does not exist or is empty
- aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
- Form element does not have an implicit (wrapped) <label>
- Form element does not have an explicit <label>
- Element has no title attribute or the title attribute is empty

See screenshot: fields in RED are failing

![image](https://user-images.githubusercontent.com/1356166/95826767-2865e300-0d76-11eb-8f54-a714bfd41bc2.png)

**Comment by tim-lai** [comment]
related: https://github.com/swagger-api/swagger-ui/pull/5298

**Comment by msftedad** [comment]
Hi Team, is there any update on this bug? Is this issue fixed?

**Comment by ho823** [comment]
Hi :) Do you need any help for this issue ?
